### DISCUSSION TITLE
Checking for ability to merge automatically…hangs and never resolves

### DISCUSSION THREAD

**Discussion by rob-yeshearing** [community/community#121015] [Repositories]
### Select Topic Area

Bug

### Body

When I open a pull request in GitHub the Checking for ability to merge automatically… is stuck and won't let me merge a PR.   Not sure how to resolve this it's also a fairly small PR so unsure why this is happening.

**UPDATE**
Bug appears to be resolved

**Comment by adam-robey** [comment]
I am seeing this, too. It was working fine yesterday, but now I'm seeing it all the time. I've tried many different pairs of branches and more than one repository.

**Reply by rob-yeshearing** [reply]
Yep was working fine for me as well and now I'm unable to merge a PR altogether.

**Reply by adam-robey** [reply]
Well, I'm glad to know that it's not just me, I guess!

**Reply by rob-yeshearing** [reply]
Yeah, let me know what comes of that support ticket hoping it's a quick resolution.

**Reply by adam-robey** [reply]
Yes, I'll definitely let you know.

**Reply by adam-robey** [reply]
I just received a boilerplate response to my support ticket, informing me that this is the result of "systemwide issues." So, they're apparently aware of it.

I'll wait a few hours before following up.

**Reply by rob-yeshearing** [reply]
awesome thanks so much!

**Reply by radove** [reply]
Command line instructions still work for those in dire need to merge until resolution can be met

**Comment by adam-robey** [comment]
I opened a support ticket a few minutes ago.

**Comment by dileepkumarmudrakola** [comment]
Same here stuck in Checking for ability to merge automatically…

**Comment by jrodshua** [comment]
+1 experiencing this issue too

**Comment by emevieira123** [comment]
Same here stuck in Checking for ability to merge automatically…

**Comment by davidritter** [comment]
Same here
+1 this issue

**Reply by davidritter** [reply]
just stated working for me

**Comment by EverthSarantes** [comment]
I also have this issue

**Comment by samanthafrances** [comment]
Having the same issue.

**Comment by radove** [comment]
I also have the same issues. Command line instructions still work for those in dire need to merge.

**Comment by sparshgup** [comment]
Same issue.

**Comment by AvdeenkoArtem** [comment]
Same

**Comment by mikegabriel** [comment]
Same issue, started about an hour ago.  Multiple repositories.

**Reply by mikegabriel** [reply]
Resolved on 2 of my PRs but still having issues on another (larger than the others) so but should clear with time.

**Comment by LrAmaral** [comment]
Same issue here

**Comment by flnh** [comment]
Same here,
I first encountered this problem about 1 hour ago.

**Comment by adam-robey** [comment]
Seems to be working again for me!

**Reply by flnh** [reply]
I comfirm it's working on my side 👍

**Comment by voriseks** [comment]
same issue here

**Reply by flnh** [reply]
Try again, it seems to be working again now

**Comment by LrAmaral** [comment]
it's working here now

**Comment by rob-yeshearing** [comment]
Still broken for me, I live in the south florida area

**Reply by Tomas00217** [reply]
Doesn't work for me either. Central Europe

**Reply by choleQuorum** [reply]
Same issue here! . Argentina

**Comment by abrahamvargas-tdo** [comment]
We started seeing this issue about 30 mins ago... Status page says an incident was fixed about 1:30 ago but looks like it wasn't completely fixed

**Reply by rob-yeshearing** [reply]
yep still hanging for me and i live in south florida

**Reply by emmyileeva** [reply]
still not working for me either

**Reply by abrahamvargas-tdo** [reply]
SoCal over here... I guess we'll have early lunch

**Reply by abrahamvargas-tdo** [reply]
Working now!

**Reply by eugene-huang-stori** [reply]
Working now for me in New York.

**Comment by Pushpit07** [comment]
Same issue

**Reply by dev-searchengineunity** [reply]
same for me

**Comment by frankmurgoloatcsulbedu** [comment]
Same issue for me here in SoCal USA

**Comment by DatChive** [comment]
Same issue - west coast. Finally got some free time between meetings and now I can't merge anything. Naturally [the status page](https://www.githubstatus.com/) shows everything working normally.

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExOWYzNGRiNG9nNDN1NG1iYzlkMTZldXVzdGJqa2J3N3Mzem56cHM4cSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/SWoRKslHVtqEasqYCJ/giphy.gif)

**Comment by noamteyssier** [comment]
Same issue for me in NorCal USA

**Comment by ashwathrabin** [comment]
Same issue here in Germany,

**Comment by jonathans199** [comment]
GH is having PR merging delays
![Screenshot 2024-06-28 at 2 51 36 PM](https://github.com/community/community/assets/30424887/271215a2-4d86-4f0c-b811-e482bbd0dbb8)

Check out https://www.githubstatus.com/

**Comment by farias-ar** [comment]
We're dealing with the same issue here in Brazil.

**Comment by Luise8** [comment]
I'm getting this now.

**Comment by naeemhaq** [comment]
Is there a resolution for this.

**Comment by denisgojak** [comment]
Same here

**Comment by hunex91** [comment]
Same here

**Comment by rreece** [comment]
Same for me
